service/log/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+exec 2>&1
+exec multilog t s25000 n4 /var/log/dbus-opendtu
```

Comment:
I propose to add some inline comment about the rather cryptic command here. At least a link to
https://manpages.debian.org/stretch/daemontools/multilog.8.en.html

Reply:
added